test/integration/consul-container/libs/cluster/container.go

```diff
@@ -59,6 +61,12 @@ func (c *consulContainerNode) ClaimAdminPort() int {
 	return p
 }
 
+func (c *consulContainerNode) ClaimConnectPort() int {
+	p := 21000 + c.nextConnectPortOffset
+	c.nextConnectPortOffset++
+	return p
+}
```

Comment:
I started going this route to bang on the public listener and generate access logs. Since I ended up not including this in the test because there is more to Envoy's internal responses than I thought, we could take or leave this.

Reply:
I decided to remove this. It's just more noise and it makes the re-ordering I did between starting the sidecar and service registration more challenging.